src/Corax/IndexSearcher/IndexSearcher.cs

```diff
@@ -338,8 +349,10 @@ public FieldMetadata GetFieldMetadata(string fieldName, FieldIndexingMode mode =
             handler.Dispose();
             return binding.Metadata;
         }
+        
+        Slice.From(Allocator, $"{fieldName}-C", ByteStringType.Immutable, out var fieldTermTotalSumField);
```

Comment:
![image](https://user-images.githubusercontent.com/86351904/218687530-27f16947-8d21-492a-be3a-b54c0908ae51.png)


This is implemented to calculate this part. Since we're working with batches etc is impossible to calculate the average so I'm storing the sum of lengths of already indexed terms and then calculating the average in runtime as `total_length / number_of_terms) `

Reply:
You should add this explanation with the formula right there in the code with a link to the actual document and the page where that comes from.